packages/utils/src/getCountryCode.js

```diff
@@ -10,13 +10,7 @@ export const getCountryCode = (countryName, entityObjects = []) => {
   }
 
   const { getCode: getCountryIsoCode } = require('countrynames');
-  // Use the country ISO code if there's a direct match,
-  //otherwise base on the two letter facility code prefix
-  const countryCode =
-    getCountryIsoCode(countryName) || entityObjects.length
-      ? entityObjects[0].code.substring(0, 2)
-      : null;
-
```

Comment:
Original reason for the line that interpolates the country code from the entity code prefix was to deal with demo land. Now that we have an explicit check for Demo Land (see a few lines up), we shouldn't have any other countries coded using non-ISO codes, so can remove this dangerous behaviour.